src/AspNet.Security.OAuth.MyobAccounting/MyobAccountingAuthenticationDefaults.cs

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNetCore.Builder;
+
+namespace AspNet.Security.OAuth.MyobAccounting {
```

Comment:
Is this OAuth2 provider specific to their "accounting" product or is their OAuth2 implementation common to all their products? If so, maybe it would be better to name it "Myob" instead of "MyobAccounting".